src/du_parameters/guard_band.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Define a dictionary where SCS values map to bandwidths and their guard bands."""
+
+import logging
+
+from src.du_parameters.frequency import Frequency
+
+logger = logging.getLogger(__name__)
+
+
+class GuardBandError(Exception):
+    """Exception raised when guard band calculation fails."""
+
+    pass
+
+
+minimum_guard_bands = {
```

Comment:
Let's indicate this one is private.

```suggestion
_minimum_guard_bands = {
```